dali/image/image_factory.cc

```diff
@@ -73,15 +78,17 @@ bool CheckIsTiff(const uint8_t *tiff, int size) {
 std::unique_ptr<Image>
 ImageFactory::CreateImage(const uint8_t *encoded_image, size_t length, DALIImageType image_type) {
   DALI_ENFORCE(CheckIsPNG(encoded_image, length) + CheckIsBMP(encoded_image, length) +
-               CheckIsGIF(encoded_image, length) + CheckIsJPEG(encoded_image, length)
-             + CheckIsTiff(encoded_image, length) == 1,
+               CheckIsGIF(encoded_image, length) + CheckIsJPEG(encoded_image, length) +
+               CheckIsPNM(encoded_image, length) == 1,
```

Comment:
I think you have removed Tiff by accident

Reply:
You are correct.  I will fix.